.github/workflows/release.yml

```
@@ -0,0 +1,30 @@
+name: Release Gem
+
+on:
+  workflow_dispatch:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
```

次のように特定のフォーマットのタグがPushされたときにリリースすると良いと思います。

```suggestion
on:
  push:
    tags:
      - 'v*'
```

利点は次の2つです。

1. リリースバージョンのソースコードがタグで特定できる
2. 毎回のPush時にはリリースしたくない。ある程度機能ができてからリリースしたい

---

タグを指定してPushしたときにリリースするように修正しました。

[Update release.yml to trigger gem release on version tag push](https://github.com/Tamada-Arino/simple-inline-text-annotation/pull/12/commits/46c760f7d6a49cf44a3d45d6fdfb84ca361754bd)